app/models/user_notification.rb

```diff
@@ -0,0 +1,20 @@
+class UserNotification < ActiveRecord::Base
+  include ApplicationNotificationsHelper
```

Comment:
Main problem is that I need to get the path in `send_email` method. And what about this solution: